src/lib/provable/wrapped-classes.ts

```diff
@@ -19,3 +19,6 @@ function Bytes(size: number) {
 Bytes.from = InternalBytes.from;
 Bytes.fromHex = InternalBytes.fromHex;
 Bytes.fromString = InternalBytes.fromString;
+
+// expore base class so that we can detect Bytes with `instanceof`
+Bytes.Base = InternalBytes;
```

Comment:
we should _always_ expose base classes somewhere for type factories, so that users can detect what class they have using `instanceof`.
putting base classes on `.Base` of the factory function seems like a nice, non-intrusive pattern to me.